metomi/isodatetime/data.py

```diff
@@ -413,11 +433,9 @@ def get_days_and_seconds(self):
         Duration which has self.seconds = CALENDAR.SECONDS_IN_DAY +
         100 will return 1 day, 100 seconds or (1, 100) from this
         method.
-
```

Comment:
Doesn't matter but FYI these trailing blank-lines in docstrings are legal - https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html